apps/json-api-server/src/app/rpc/service/rpc.service.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  InputType,
+  OutputType,
+  RpcService as IRpcService,
+} from '@nestjs-json-api/type-for-rpc';
+
+import {
+  createErrorCustomError,
+  RpcHandler,
+  RpcParamsPipe,
+} from '@klerick/nestjs-json-rpc';
+import { ParseIntPipe } from '@nestjs/common';
+
+@RpcHandler()
+export class RpcService implements IRpcService {
+  methodeWithObjectParams(a: InputType): Promise<OutputType> {
```

Comment:
typo in word "method", in some other places as well. can do word search by "methode" and correct to "method"